src/d3a/models/read_user_profile.py

```diff
@@ -294,18 +286,18 @@ def _eval_time_period_consensus(input_profile: Dict):
 
 
 def time_str(hour, minute):
-    return "{:02d}{:02d}".format(hour, minute)
+    return f"{hour}:{minute}"
```

Comment:
Since it is a string anyways, why not adhering to the former format ?

Reply:
you mean `f"{hour:02d}:{minute:02d}"` right? 
Agreed!